main.go

```diff
@@ -68,6 +60,27 @@ func main() {
 	}
 }
 
+// Install custom signal handlers:
+func installSignalHandlers(ctx context.Context, m *firecracker.Machine) {
+	go func() {
+		// Clear the INT handler installed by the firecracker SDK:
+		signal.Reset(os.Interrupt, syscall.SIGTERM, syscall.SIGQUIT)
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+
+		for {
+			switch s := <-c; {
+			case s == syscall.SIGTERM || s == os.Interrupt:
+				log.Printf("Caught SIGINT, requesting clean shutdown")
+				m.Shutdown(ctx)
+			case s == syscall.SIGQUIT:
```

Comment:
Will `SIGQUIT` ever be sent to the channel? I noticed `Notify` doesn't explicitly have it in its parameters. Is that due to `SIGTERM` also includes `SIGQUIT`?

Reply:
Thanks, that was an oversight.

In reality, we don't need to handle SIGQUIT here at all, since the SDK implements the same SIGQUIT handling internally. But since there's some disagreement about whether it should do that or not, I think it's good to be explicit.